testing/python/fixtures.py

```diff
@@ -4207,3 +4207,32 @@ def test_bug(value):
     )
     result = testdir.runpytest()
     result.assert_outcomes(passed=10)
+
+
+def test_fixture_arg_ordering(testdir):
```

Comment:
```suggestion
def test_fixture_arg_ordering(testdir):
    """
    This test describes how fixtures in the same scope but without explicit dependencies
    between them are created. While users should make dependencies explicit, often
    they rely on this order, so this test exists to catch regressions in this regard.
    See #6540 and #6492.
    """ 
```

Reply:
@nicoddemus 
Thanks, please force-push your suggestion and merge then.

Reply:
Done. 👍